scss/mixins/_text-hide.scss

```diff
@@ -1,11 +1,13 @@
 // CSS image replacement
-@mixin text-hide() {
+@mixin text-hide($ignore-warning: false) {
   // stylelint-disable-next-line font-family-no-missing-generic-family-keyword
   font: 0/0 a;
   color: transparent;
   text-shadow: none;
   background-color: transparent;
   border: 0;
 
-  @warn "The `text-hide()` mixin has been deprecated as of v4.1.0. It will be removed entirely in v5.";
+  @if ($ignore-warning == false) {
```

Comment:
I'm thinking maybe checking against `true` would be better? Since that's what we use, and we can't know what the user might use.

Reply:
I always prefer to be as strict as possible.